### ISSUE TITLE
New BR for BT-131

### ISSUE THREAD

**Issue by rkottmann** [itplr-kosit/xrechnung-schematron#20]
It would be very good to implement this rule:

 
BT-131 = BT-129 * BT-146 – BT-136 + BT-141

In German words:
Gesamtpreis (netto) = Menge * Preis pro Einheit (netto) inkl. Rabatt – Nachlass (netto) + Zuschlag (netto)

**Comment by rkottmann** [comment]
Please only as Information and not required rule

**Comment by rkottmann** [comment]
@mircokuhlmann can we start on this please?

**Comment by mircokuhlmann** [comment]
@rkottmann Of course! I am waiting for feedback to my schematron proposal in https://projekte.kosit.org/xrechnung/xrechnung/issues/22.
